action.yml

```diff
@@ -175,3 +193,9 @@ runs:
         if [[ -f __inputs.yml ]]; then
           rm __inputs.yml
         fi
+        if [[ -f ${{ github.action_path }}/composer.lock ]]; then
+          rm ${{ github.action_path }}/composer.lock
+        fi
+        if [[ -d ${{ github.action_path }}/vendor ]]; then
+          rm -r ${{ github.action_path }}/vendor
```

Comment:
```suggestion
          rm -rf ${{ github.action_path }}/vendor
```

Reply:
Shouldn't be necessary since we're not doing a `--prefer-source` - but done nevertheless since it doesn't hurt anything I think.